instrumentation/pg/lib/opentelemetry/instrumentation/pg/version.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module PG
+      VERSION = '0.16.0'
+      MIN_PG_GEM_VERSION = '1.1.0'
```

Comment:
Minor nitpick, previously we have put this definition in the base instrumentation class that invokes it.  Whether or not that is the right place for it I'd like to keep that consistence even if we end up moving them all here later.

Reply:
@robertlaurin No worries - I made that change in 409a7d09 . I think I had originally put it in `version.rb` to re-use it in the gemspec, but I agree that the overall consistency is nicer. It's not like we expect it to change very often, so my original location was optimizing for the wrong thing! 😄